--- a/luci-app-daed/root/etc/init.d/luci_daed
+++ b/luci-app-daed/root/etc/init.d/luci_daed
@@ -1,6 +1,6 @@
 #!/bin/sh /etc/rc.common

-START=98
+START=99
 USE_PROCD=1

@@ -14,7 +14,6 @@ start_service() {
        procd_set_param stdout 1
        procd_set_param stderr 1
        procd_close_instance
-       hijack_resolv_conf
 }
 
 stop_service() {
@@ -22,5 +21,4 @@ stop_service() {
        procd_kill $NAME
        rm -f /tmp/daed.lock
        rm -f /tmp/daed.pid
-       restore_resolv_conf
 }
